src/com/amazon/paapi5/v1/BrowseNode.php

```diff
@@ -401,7 +401,7 @@ public function setSalesRank($salesRank)
      *
      * @return boolean
      */
-    public function offsetExists($offset)
+    public function offsetExists(mixed $offset): bool
```

Comment:
Was this type hinting done as part of the code sniffing you mentioned?

Reply:
Yep, we needed to add the typehints for the various [ArrayAccess](https://www.php.net/manual/en/class.arrayaccess.php) methods as they were presenting a Deprecation Warning before this.